.gitignore

```diff
@@ -98,3 +98,4 @@ coverage/
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+!Gemfile.lock
```

Comment:
super nit: May want to be `dev/ci/**/Gemfile.lock`. If we did start using Gemfiles in other places in the repo besides CI I don't think we would want to commit them. I don't know why we would elsewhere though.

Reply:
done.